networking/external_dns_operator/external-dns-operator-release-notes.adoc

```diff
@@ -0,0 +1,72 @@
+// ExternalDNS Operator Release Notes
+:_mod-docs-content-type: ASSEMBLY
+[id="external-dns-operator-release-notes"]
+= ExternalDNS Operator release notes
+:context: external-dns-operator-release-notes
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+The ExternalDNS Operator deploys and manages `ExternalDNS` to provide name resolution for services and routes from the external DNS provider to OpenShift Container Platform.
+
+These release notes track the development of the ExternalDNS Operator in {product-title}.
+
+[id="external-dns-operator-release-notes-1.2.0"]
+== ExternalDNS Operator 1.2.0
+
+The following advisory is available for the ExternalDNS Operator version 1.2.0:
+
+* link:https://access.redhat.com/errata/RHEA-2023:7239[RHEA-2022:5867 ExternalDNS Operator 1.2 operator/operand containers]
+
+[id="external-dns-operator-1.2.0-new-features"]
+=== New features
+
+The ExternalDNS Operator now supports shared VPC.
```

Comment:
Eacn new feature should be its own bullet point in this section.

Reply:
mistake was made, one should be a bug fix